beluga-aws-sdk/src/jobs/datetime.rs

```diff
@@ -0,0 +1,40 @@
+use std::ops::Add;
+use std::time::Duration;
+
+use chrono::prelude::*;
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Deserializer, Serializer};
+
+pub(super) fn serialize<S>(date: &Option<DateTime<Utc>>, serializer: S) -> Result<S::Ok, S::Error>
```

Comment:
Can you add the unit tests from https://github.com/inomotech/iot-humpback/blob/890793e85576b08ef71656beedfaa7b085a27336/packages/hb-core/src/iot_core/client.rs#L146 as well?